research/slim/nets/cyclegan.py

```diff
@@ -0,0 +1,269 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
We have more goodies than just NASnet =D. Maybe add these to your commit message.